firebase_admin/_messaging_utils.py

```diff
@@ -57,6 +57,9 @@ def __init__(self, data=None, notification=None, android=None, webpush=None, apn
         self.topic = topic
         self.condition = condition
 
+    def __str__(self):
+        return json.dumps(MessageEncoder().default(self))
```

Comment:
Would this work?

```
json.dumps(self, cls=MessageEncoder)
```

Reply:
Yes! Thank you.
Will update with test cases soon.